src/server/main_service.cc

```diff
@@ -1172,6 +1172,8 @@ std::optional<ErrorReply> Service::VerifyCommandState(const CommandId* cid, CmdA
 }
 
 void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx) {
+  absl::Cleanup clear_last_error(
```

Comment:
why do we need this here? what are the cases that we ignore?

Reply:
what do you mean these are not interesting for us?
we want to see error messages for commands sent to dragonfly, so if we have problem we will know the command and the params sent and it will be easy to reproduce the problem and to know if this is bug on our side or on client side.